crates/sui-graphql-rpc/src/types/object.rs

```diff
@@ -1214,11 +1210,11 @@ impl RawPaginated<Cursor> for StoredHistoryObject {
 }
 
 impl Target<Cursor> for StoredHistoryObject {
-    fn cursor(&self) -> Cursor {
-        Cursor::new(HistoricalObjectCursor {
-            object_id: self.object_id.clone(),
-            checkpoint_viewed_at: self.checkpoint_sequence_number as u64,
-        })
+    fn consistent_cursor(&self, checkpoint_viewed_at: u64) -> Cursor {
```

Comment:
If there is only `consistent_cursor` now, can we rename this back to just `cursor` as a follow up refactor?